plugin/trino-neo4j/src/main/java/io/trino/plugin/neo4j/Neo4jJdbcConfig.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.neo4j;
+
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+
+public class Neo4jJdbcConfig
```

Comment:
In this case can we use `BaseJdbcConfig` directly ?

Reply:
Yes, that should work

Reply:
Removed